crates/router/src/compatibility/stripe/payment_intents/types.rs

```diff
@@ -266,9 +344,9 @@ impl TryFrom<StripePaymentIntentRequest> for payments::PaymentsRequest {
             mandate_data: mandate_options,
```

Comment:
mandate_data mapping here is using mandate_options variable which is being derived from ForeignTryFrom trait, i can't find the `impl From<MandateData> for payments::MandateData` being used.

Reply:
Will add that